src/views/pages/name/private-limited-partnership-name-scotland.njk

```diff
@@ -0,0 +1,104 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <form class="form" action={{ props.currentUrl }} method="post">
+
+      <input type="hidden" name="pageType" value={{ props.pageType }}>
+
+      {% include "includes/csrf_token.njk" %}
+
+      <h1 class="govuk-heading-xl">{{ i18n.privateNameScottishPage.title }}</h1>
+
+      <div class="govuk-form-group">
+        <h1 class="govuk-label-wrapper">
+          <label class="govuk-label govuk-label--l" for="partnership_name">
+            {{ i18n.privateNameScottishPage.whatIsName }}
+          </label>
+        </h1>
+
+        {{ govukInput({
+          classes: "govuk-input--width-40",
+          errorMessage: props.errors.private_partnership_scotland_name if errors,
```

Comment:
errorMessage: props.errors.partnership_name if errors,

this is the name of the field
we don't store the name in different ways depending on the subpage

Reply:
Fixed now.